vcpkg.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "main",
+  "version-string": "latest",
```

Comment:
`latest` is no good from the security perspective

Reply:
Done, but not pushed out.